script/zlib.decompress.py

```diff
@@ -0,0 +1,20 @@
+import zlib
+import sys
+
+CHUNKSIZE = 1024
+zlib_obj = zlib.decompressobj(int(sys.argv[2]))
```

Comment:
dal bych sem validaci na pocet parametru, kdyby se neco vydrbalo, tak at se to vcas zastavi a neplive to nejaky undefined index